src/main/java/mate/academy/controller/BookController.java

```diff
@@ -30,7 +32,7 @@ public List<BookDto> getAll() {
     }
 
     @PostMapping
-    public BookDto save(@RequestBody CreateBookRequestDto requestDto) {
+    public BookDto save(@RequestBody @Valid CreateBookRequestDto requestDto) {
```

Comment:
What about validation update dto? 
Also, you can add `@Positive Long id` where is needed and add `@Validated` under the controller